### ISSUE TITLE
Normalization of the noise input

### ISSUE THREAD

**Issue by htzheng** [rosinality/stylegan2-pytorch#228]
I notice that in the tensorflow/pytorch official code, the input noise z is normalized (https://github.com/NVlabs/stylegan2/blob/315ac5494112c94f3af5902d7c492b4dc471d112/training/networks_stylegan2.py#L283 and https://github.com/NVlabs/stylegan2-ada-pytorch/blob/d4b2afe9c27e3c305b721bc886d2cb5229458eba/training/networks.py#L220) I wonder if the input noise normalization supported in the current implementation? Many thanks!

**Comment by rosinality** [comment]
You can refer to this: https://github.com/rosinality/stylegan2-pytorch/blob/master/model.py#L407

**Comment by htzheng** [comment]
Thanks a lot!
